src/cltl/triple_extraction/api.py

```diff
@@ -525,123 +492,4 @@ def _clean(tokens):
-        tokenized_sentence = utterance.tokens
-        pos = self.POS_TAGGER.tag(tokenized_sentence)  # standford
-        alternative_pos = pos_tag(tokenized_sentence)  # nltk
-
-        self._log.debug(pos)
-        self._log.debug(alternative_pos)
-
-        if pos != alternative_pos:
-            self._log.debug('DIFFERENT POS tag: %s != %s' % (pos, alternative_pos))
-
-        # fixing issues with POS tagger (Does and like)
-        ind = 0
-        for w in tokenized_sentence:
-            if w == 'like':
-                pos[ind] = (w, 'VB')
-            ind += 1
-
-        if pos and pos[0][0] == 'Does':
-            pos[0] = ('Does', 'VBD')
-
-        # the POS tagger returns one tag with a $ sign (POS$) and this needs to be fixed for the CFG parsing
-        ind = 0
-        for word, tag in pos:
-            if '?' in word:
-                word = word[:-1]
-            if tag.endswith('$'):
-                new_rule = tag[:-1] + 'POS -> \'' + word + '\'\n'
-                pos[ind] = (pos[ind][0], 'PRPPOS')
-            else:
-                # CFG grammar is created dynamically, with the terminals added each time from the specific utterance
-                new_rule = tag + ' -> \'' + word + '\'\n'
-            if new_rule not in self._cfg:
-                self._cfg += new_rule
-            ind += 1
-
-        try:
-            cfg_parser = CFG.fromstring(self._cfg)
-            rd = RecursiveDescentParser(cfg_parser)
-
-            last_token = tokenized_sentence[len(tokenized_sentence) - 1]
-
-            if '?' in last_token:
-                tokenized_sentence[len(tokenized_sentence) - 1] = last_token[:-1]
-
-            parsed = rd.parse(tokenized_sentence)
-
-            s_r = {}  # syntactic_realizations are the topmost branches, usually VP/NP
-            index = 0
-
-            forest = [tree for tree in parsed]
-
-            if len(forest):
-                if (len(forest)) > 1:
-                    self._log.debug('* Ambiguity in grammar *')
-                for tree in forest[0]:  # alternative trees? f
-                    for branch in tree:
-                        s_r[index] = {'label': branch.label(), 'structure': branch}
-                        raw = ''
-                        for node in branch:
-                            for leaf in node.leaves():
-                                raw += leaf + '-'
-
-                        s_r[index]['raw'] = raw[:-1]
-                        index += 1
-            else:
-                self._log.debug("no forest")
-
-            for el in s_r:
-                if type(s_r[el]['raw']) == list:
-                    string = ''
-                    for e in s_r[el]['raw']:
-                        string += e + ' '
-                    s_r[el]['raw'] = string
-                s_r[el]['raw'] = s_r[el]['raw'].strip()
-
-            return forest, s_r
-        except:
-            return [], {}
+        return '{:10s} {:03d}: "{}"'.format(author, self.turn, self.transcript)
```

Comment:
I moved the [info](https://github.com/leolani/cltl-knowledgeextraction/pull/4/files/e154c8f8d8f3a51128054a0d8e36c5081b0ef222#r786514735) from the Chat log to the Utterance log